src/qililab/calibration/calibration_node.py

```diff
@@ -452,7 +452,7 @@ def run_node(self, check: bool = False) -> float:
 
         # When notebook execution fails, generate error folder and move there the notebook:
         except Exception as exc:  # pylint: disable = broad-exception-caught
-            if output_path in [os.scandir(self.nb_folder)]:
+            if output_path in [os.getcwd()]:
```

Comment:
Why is this a `in list`, instead than a` ==` ? 👀 

(Before we had a list, not anymore)

Reply:
True it scan the directory again, I just changed it!